### PR TITLE
SAK-40517. Changes to show linear access message only when this optio…

### PR DESCRIPTION
**Pull request by mateullas**
…n is selected. The message itself is also clarified.

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp [hunk 1]
```diff
@@ -84,13 +84,15 @@
               }
 
               if (enabled) {
+            	  $('#assessmentSettingsAction\\:linear_access_warning').hide();
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:87]: Indentation

File: samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp [hunk 2]
```diff
@@ -84,13 +84,15 @@
...
                   $('#assessmentSettingsAction\\:markForReview1').parent().removeClass("placeholder");
                   QuesFormatRadios.forEach( function(v, i, a) {
                       $('label[for="' + v + '"]').removeClass("placeholder");
                       $("#" + v).removeAttr("disabled");
                   });
               } else {
+            	  $('#assessmentSettingsAction\\:linear_access_warning').show();
                   $('#assessmentSettingsAction\\:markForReview1').attr("disabled", true);
                   $('#assessmentSettingsAction\\:markForReview1').prop("checked", false);
                   $('#assessmentSettingsAction\\:markForReview1').parent().addClass("placeholder");
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:98]: I know this is how it was before, but you could just chain all these calls onto the one jquery object.

Comment (mateullas) [reply samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:98]: Hi @adrianfish I'm not sure I understand what you mean. 
Which calls do you refer? Do you refer to chain calls to "markForReview1"? 
I have introduced just the "linear_access_warning" lines, and there is one call onlly

Comment (adrianfish) [reply samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:98]: Yes, just to clean up the code a bit and make it a bit performant. I know it wasn't your code, but it's always nice to do a bit of cleanup around the area you're fixing.

$('#assessmentSettingsAction\\:markForReview1')
  .attr("disabled", true)
  .prop("checked", false)
  .parent().addClass("placeholder");

**Pull request closed without merging**